StyleCop.Analyzers/StyleCop.Analyzers/Helpers/IndentationOptions.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace StyleCop.Analyzers.Helpers
+{
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Formatting;
+    /// <summary>
+    /// Class used for obtaining information about indentation options.
+    /// </summary>
+    internal class IndentationOptions
+    {
+        private IndentationOptions(Workspace workspace)
+        {
+            var options = workspace.Options;
+            this.IndentationSize = options.GetOption(FormattingOptions.IndentationSize, LanguageNames.CSharp);
+            this.TabSize = options.GetOption(FormattingOptions.TabSize, LanguageNames.CSharp);
+            this.UseTabs = options.GetOption(FormattingOptions.UseTabs, LanguageNames.CSharp);
+        }
+        /// <summary>
```

Comment:
There are spacing missing between all elements in this class. This will trigger SA1516.